modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+// * microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-dry-run-v2_{context}"]
+= Performing a dry run for V2
+
+You can use oc-mirror to perform a dry run, without actually mirroring any images. This allows you to review the list of images that would be mirrored. A dry run also allows you to catch any errors with your image set configuration early. 
+
+.Procedure
+
+. Run the `oc mirror` command with the `--dry-run` flag to perform a dry run:
++
+[source,terminal]
+----
+$ oc mirror --config <imageset-Config.yaml> <1>
```

Comment:
@subhtk command should be `oc-mirror --config <imageset-config.yaml> file://<oc-mirror workspace path> --v2` do not see `--v2` which is important. Also dry-run for v2 behaves a little differently. you can read and capture about this from the doc here https://docs.google.com/document/d/1NoRR_pG4W10Q1KiOZ_LaCUuYuAxgVlrv_4BPu-6K-5s/edit#heading=h.vmuf853hmqmh Question no:6